packages/@aws-cdk/aws-apprunner/lib/service.ts

```diff
@@ -430,9 +467,24 @@ export interface ImageConfiguration {
    * Environment variables that are available to your running App Runner service.
    *
    * @default - no environment variables
+   * @deprecated use environmentVariables()
```

Comment:
```suggestion
   * @deprecated use environmentVariables
```

Looks like you're not setting this variable anymore. That means anyone currently reading from it today will find an empty environment in the next release? It should contain the same values as `environmentVariables`.

Also by the way: I don't really like that this is optional. We can always put an empty object `{}` in there if there were no variables to begin with.